src/components/layout/dashboard/contextHeader/useDashboardBreadcrumbs.tsx

```diff
@@ -224,6 +224,9 @@ export const useAdminBreadcrumbConfig = (): CrumbConfig => {
       [AdminDashboardRoutes.CLIENT_MERGES]: {
         title: 'Merge Clients',
       },
+      [AdminDashboardRoutes.USERS]: {
+        title: 'Users',
+      },
```

Comment:
fwiw, I don't _think_ this is necessary -- it should default to camelized version of the route name. It's only necessary if the breadcrumb title differs, or if you need to establish a hierarchy with another page